src/GoogleClient.js

```diff
@@ -193,7 +195,7 @@ export class GoogleClient {
         : 'and mimeType != \'application/vnd.google-apps.folder\'';
     }
 
-    const opts = {
```

Comment:
in the Google API doc, this argument is called `paramsOr...` - renamed to not make it too confusing with `googleApiOpts`